tools/commit.template

```diff
@@ -0,0 +1,43 @@
+
+
+# Commit messages are very important to us,
+# which is why we have a strict commit message policy in place.
+# Please use the following guidelines to format all your commit
+# messages:
+#
+#     <type>: <subject>
+#     <BLANK LINE>
+#     <body>
+#     <BLANK LINE>
+#     <footer>
+#
+# Please note that:
+#  - The HEADER is a single line of max. 72 characters that
+#    contains a succinct description of the change. It contains a
+#    type and a subject.
+#
+#      + <type> describes the kind of change that this commit is
+#               providing. Allowed types are:
+#             * feat (feature)
+#             * deps (dependency updates)
+#             * fix (bug fix)
+#             * docs (documentation)
+#             * package (ignore files, CI/CD)
+#             * style (formatting, missing semicolons, …)
+#             * refactor
+#             * test (when adding missing tests)
+#             * revert (a reversion commit)
+#             * WIP (work in progress, ie. in development)
+#
+#       + <subject> is a very short description of the change, in
+#                   the following format:
+#             * Imperative, present tense: “change” not
+#               “changed”/“changes”
+#             * Upper and lower case letters acceptable
+#             * No dot (.) at the end
+#
+#  - The BODY should include a detailed description of the changes and
+#    contrast this with previous behavior and must be phrased in
+#    imperative present tense. Minimum 20, max 72 characters.
+#  - The FOOTER is optional, hut can contain information such as
+#    `Semver: patch|minor|major`, `Ref: <Jira>`, and `Fixes: <link>`
```

Comment:
`Jira` doesn't make sense here anymore. Should be a valid github issue reference